tutorials/boundary_conditions.ipynb

```diff
@@ -0,0 +1,440 @@
+{
```

Comment:
<p>Please add "a":</p><p>We can directly set the values of F on a boundary by:</p><p> </p><p>Could we change the order of sentences starting with "Pro tip" and "We used...". I think that makes it clearer what "this type" refers to.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/1073/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>